blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -140,15 +147,15 @@ OrganizationPipelines.propTypes = {
     params: object, // From react-router
     children: node, // From react-router
     location: object, // From react-router
-    pipelines: array,
+    allPipelines: array,
+    organizationPipelines: array,
 };
 
 OrganizationPipelines.childContextTypes = {
     pipelines: array,
-    pipeline: object,
 };
 
-const selectors = createSelector([pipelinesSelector],
-    (pipelines) => ({ pipelines }));
+const selectors = createSelector([allPipelinesSelector, organizationPipelinesSelector],
+    (allPipelines, organizationPipelines) => ({ allPipelines, organizationPipelines }));
```

Comment:
@michaelneale well, the dashboard would, in theory show you all pipelines you have access to, while the org view would only show you pipelines within the org. The fetches and paging would be separate. Since I had to change this around due to paging, anyway, I figured best to go ahead and build it in a way that makes the most sense. Though, right now with the "jenkins" org, it's very awkward ever seeing the org page at all and there are probably still enough things hardcoded that it will not work (I tried to replace the /jenkins references in the rest calls with org where I found them, but I don't think I got them all), anyway, just one thing we don't have to completely redo when we get to making orgs functional